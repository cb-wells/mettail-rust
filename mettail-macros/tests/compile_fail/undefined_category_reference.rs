use mettail_macros::theory;

theory! {
    name: Invalid,
    
    exports {
        Proc
    }
    
    terms {
        // Error: references undefined 'Name' category
        Drop . Proc ::= "*" (Name) ;
    }
}

fn main() {}

