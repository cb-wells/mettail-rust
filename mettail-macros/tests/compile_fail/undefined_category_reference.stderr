error: expected identifier
  --> tests/compile_fail/undefined_category_reference.rs:12:29
   |
12 |         Drop . Proc ::= "*" (Name) ;
   |                             ^
